api/v1/metric_routes_test.go

```diff
@@ -26,7 +26,12 @@ import (
 	"net/http/httptest"
 	"testing"
 
+	"github.com/stretchr/testify/require"
+
+	"github.com/sirupsen/logrus"
```

Comment:
Can we have an issue to add `logrus` to the whole codebase ? As I much prefer it this way than to be aliased to log.

Reply:
I agree, I've changed it in a bunch of places, but a lot of the old imports still remain...
https://github.com/loadimpact/k6/issues/1016